README.md

```diff
@@ -74,36 +74,65 @@ ETH 0x0000000000000000000000000000000000000000 Balance: 2
 Now if you navigate back to your browser (http://localhost:3000/#/YOUR-DAO-ADDRESS) you'll be able to open the Redemptions app and add one of these tokens to your locally deployed Redemptions app. 
 
 
+## Using Redemptions
 
-## How to deploy Redemptions to an organization
+The redemptions app allows organizations to add and remove tokens from a list of eligible tokens. When a user choses to redeem tokens they will receive a proportional share of all eligible tokens in the `Vault`.
 
-Redemptions has been published to APM on rinkeby at `redemptions.open.aragonpm.eth`
+The tokens you can add to the Redemptions app are dependent on the blockchain you're on. If you're on the Ethereum mainnet you can add any erc-20 token contract. If you're on a local testnet you can add any token contracts that have been deployed locally. 
 
-To deploy to an organization you can use the [Aragon CLI](https://hack.aragon.org/docs/cli-intro.html).
+Who/what is allowed to control the Redemptions app is determined by the DAO's permissions. You can check the permissions for your DAO in the `Permissions` tab of the Aragon client. Every app currently installed should be list. Click on an app to view it's permissions. 
+- Note: the section you're looking for is "Permissions set on this app", not the permissions available or permissions granted. The "Permissions set on this app" section will tell you who/what can do stuff with the app currently.
 
-```sh
-aragon dao install <dao-address> redemptions.open.aragonpm.eth --app-init-args <vault-address> <token-manager-address>
-```
+<p align="center">
+    <img src="https://raw.githubusercontent.com/1Hive/redemptions-app/master/docs/resources/redemptions-app-permissions.gif" width="600" />
+</p>
 
-The Redemptions app must have the `TRANSFER_ROLE` permission on `Vault` and the `BURN_ROLE` permission on the `Token Manager`.
+### Adding tokens:
 
+The default recommendation is for the Voting app to control adding and removing tokens from the Redemptions app. This way your community can collectively manage the state of Redemptions. Adding tokens to Redemptions via the voting app requires two steps:
+- creating a vote to add the token to the Redeptions app
+- approving the vote to add the token to the Redemptions app
 
-## Using Redemptions
+<p align="center">
+    <img src="https://raw.githubusercontent.com/1Hive/redemptions-app/master/docs/resources/add-token-to-redemptions-app.gif" width="600" />
+</p>
```

Comment:
If i recall correctly, initially @lkngtn added giffs for adding a removing tokens but in the end decided to remove them as it made the doc too heavy.

I think that with a step by step description like above should do for adding tokens but would keep the current step of clicking on `Add token`, entering token address..

Reply:
Gotcha. So the README is going to be like a quickstart and then more detailed user guides and stuff can live in the `docs` directory?